model/constants.py

```diff
@@ -0,0 +1,31 @@
+#
+# This file is part of Reclaimer.
+#
+# For authors and copyright check AUTHORS.TXT
+#
+# Reclaimer is free software under the GNU General Public License v3.0.
+# See LICENSE for more information.
+#
+
+# There is only 8 spaces allocated for these in the object's gamestate.
+# If you have more and it updates the state for those it would write into
+# unrelated fields.
+HALO_1_MAX_REGIONS = 8
+HALO_1_NAME_MAX_LEN = 31
+# These two numbers are the same because there shouldn't be more materials than
+# geometries. A geometry can only have one material after all.
+HALO_1_MAX_GEOMETRIES_PER_MODEL = 256
+HALO_1_MAX_MATERIALS = HALO_1_MAX_GEOMETRIES_PER_MODEL
```

Comment:
Make this explicitly also 256. The number of materials isn't tied to the number of geometries as far as we know.